src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/PublicationReleasePage.test.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+
+import PublicationReleasePage from '@frontend/modules/find-statistics/PublicationReleasePage';
+import { Release } from '@common/services/publicationService';
+import OfficialStats from './__data__/content.api.response.official.stats.json';
+import NationalStats from './__data__/content.api.response.national.stats.json';
+
+jest.mock('next/router', () => {
+  return {
+    push: () => {},
+    prefetch: () => {},
+  };
+});
+
+describe('PublicationReleasePage', () => {
+  test('renders national statistics image', () => {
+    const { container } = render(
+      <PublicationReleasePage
+        publication=""
+        release=""
+        data={(NationalStats as unknown) as Release}
+      />,
+    );
+
+    expect(
+      container.querySelector(
+        'img[alt="UK statistics authority quality mark"]',
+      ),
+    ).toBeDefined();
+  });
+
+  test('renders national statistics section', () => {
+    const { container } = render(
+      <PublicationReleasePage
+        publication=""
+        release=""
+        data={(NationalStats as unknown) as Release}
+      />,
+    );
+
+    // @ts-ignore
```

Comment:
Can this be written as:

```
    const elements = container.querySelectorAll('.govuk-accordion__section-header h3 button') as Element[],
```

without the need for the "ts-ignore"?

Reply:
The value returned from querySelectorAll doesn't have the find method on it, it's not an array.

All the elements are added to an array using the array destructuring operator, but our TypeScript configuration doesn't like this, even though it's valid to do it this way.